server/src/templates/description-live.html

```diff
@@ -0,0 +1,12 @@
+{% if model.numFeatures > 9 %}
```

Comment:
The details of this need adding to the acceptance criteria, for scrutiny and testing